crates/cxx-qt-gen/test_outputs/passthrough_and_naming.h

```diff
@@ -70,7 +70,7 @@ QPushButton_clickedConnect(
 
 namespace mynamespace::rust::cxxqtgen1 {
 ::QMetaObject::Connection
-ExternObject_dataReadyConnect(
+ExternObjectCpp_dataReadyConnect(
```

Comment:
assume the name change was intentional as looks like it was using the wrong name before? (eg it was using the rust name and not the possible cxx name)